app/components/job-tabs.js

```diff
@@ -1,11 +1,24 @@
+import { A } from '@ember/array';
+import { computed } from '@ember/object';
 import { isEmpty } from '@ember/utils';
 import Component from '@ember/component';
 
 export default Component.extend({
-
   tagName: 'div',
   classNames: ['travistab'],
 
+  messagesMaxLevel: computed('this.job.build.request.messages.@each.level', function () {
+    let msgs = A(this.get('request.messages')).rejectBy('level', 'info');
```

Comment:
Looks like `this.request` does not exist anymore. So `msgs` will always be empty. 
```suggestion
    let msgs = A(this.get('job.build.request.messages')).rejectBy('level', 'info');
```